index.js

```diff
@@ -1,10 +1,23 @@
+import * as dotenv from 'dotenv';
 import express from 'express';
+import postgres from 'postgres';
+
+dotenv.config();
 
 const app = express();
 const port = 3000;
 
-app.get('/', (req, res) => {
-  res.send('Hello World!');
+const sql = postgres({
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  database: process.env.DB_DATABASE,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+});
+
+app.get('/', async (req, res) => {
+  const [ {'?column?': one} ] = await sql`select 1;`;
```

Comment:
Hi Ben everything looks good on my end. Can you go into more detail about this line? Since we have no schema yet, we can only test the connection by asking the server to return 1?

Reply:
Yep, pretty much. This line just makes sure we can execute some SQL from the Node app